xwiki-platform-core/xwiki-platform-ckeditor/xwiki-platform-ckeditor-plugins/src/main/webjar/xwiki-table/plugin.js

```diff
@@ -69,6 +69,16 @@
     ['txtBorder', 'txtWidth', 'txtCellSpace', 'txtCellPad'].forEach(function(fieldId) {
       delete infoTab.get(fieldId)['default'];
     });
+    // Set a new default for the Headers value
+    infoTab.get( 'selHeaders' )[ 'default' ] = 'row';
+    // Create a warning message for the accessibility without headers
+    // We navigate the structure of the macro to insert this message exactly next to the header.
+    infoTab.elements[0].children[0].children.splice(4, 0, {
+      type : 'html',
+      html : '<span class="box warningmessage">' + 
+        editor.localization.get('xwiki-table.header.accessibilitywarning') + 
```

Comment:
```suggestion
        editor.localization.get('xwiki-table.header.accessibilityWarning') + 
```

From https://dev.xwiki.org/xwiki/bin/view/Community/L10N/Conventions/

> Individual parts of the key should use **camelCase** if more than one word is needed to identify the element.

Moreover, I'm not comfortable with the fact that the translation value is not escaped. One option is to do something like:

```
const warningMessage = document.createElement('span');
warningMessage.classList.add('box', 'warningmessage');
warningMessage.textContent = editor.localization.get('xwiki-table.header.accessibilitywarning');
// then use warningMessage.outerHTML
```

Reply:
I updated it in https://github.com/xwiki/xwiki-platform/pull/3310/commits/151a24f0743e14846e25eb3a453bcd4555bfff70 :+1: 
Tested on my local instance, it's building and working as expected.

CKEditor has an escape tool, `htmlEncode`. From what I understand, we can use it as an escapetool. We already use it in a similar way in https://github.com/xwiki/xwiki-platform/blob/7c1fa96c4550e58f1cdb0e4ae0957a6b73f19780/xwiki-platform-core/xwiki-platform-ckeditor/xwiki-platform-ckeditor-plugins/src/main/webjar/xwiki-link/plugin.js#L528 

$${\color{lightgreen}ADDRESSED}$$